multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -160,45 +159,30 @@ coreos:
         ExecStartPre=/usr/bin/curl http://127.0.0.1:8080/version
         ExecStart=/opt/bin/install-kube-system
 
-{{ if .UseCalico }}
-    - name: install-calico-system.service
-      command: start
-      content: |
-        [Unit]
-        Requires=kubelet.service docker.service
-        After=kubelet.service docker.service
-
-        [Service]
-        Type=simple
-        StartLimitInterval=0
-        Restart=on-failure
-        ExecStartPre=/usr/bin/curl http://127.0.0.1:8080/version
-        ExecStart=/opt/bin/install-calico-system
-{{ end }}
-
 write_files:
   - path: /opt/bin/install-kube-system
     permissions: 0700
     owner: root:root
     content: |
-      #!/bin/bash -e
-      /usr/bin/curl  -H "Content-Type: application/json" -XPOST \
-      -d @"/srv/kubernetes/manifests/kube-dns-rc.json" \
-      "http://127.0.0.1:8080/api/v1/namespaces/kube-system/replicationcontrollers"
-
-      /usr/bin/curl  -H "Content-Type: application/json" -XPOST \
-      -d @"/srv/kubernetes/manifests/kube-dashboard-rc.json" \
-      "http://127.0.0.1:8080/api/v1/namespaces/kube-system/replicationcontrollers"
-
-      /usr/bin/curl  -H "Content-Type: application/json" -XPOST \
-      -d @"/srv/kubernetes/manifests/heapster-de.json" \
-      "http://127.0.0.1:8080/apis/extensions/v1beta1/namespaces/kube-system/deployments"
-
-      for manifest in {kube-dns,heapster,kube-dashboard}-svc.json;do
-          /usr/bin/curl  -H "Content-Type: application/json" -XPOST \
-          -d @"/srv/kubernetes/manifests/$manifest" \
-          "http://127.0.0.1:8080/api/v1/namespaces/kube-system/services"
-      done
+      #!/bin/bash
+      set -euo pipefail
+
+      uuid_file_save="/var/run/coreos/rkt-kubectl-${RANDOM}.uuid"
+      function cleanup {
+        sudo rkt rm --uuid-file="$uuid_file_save"
+        sudo rm "$uuid_file_save"
+      }
+
+      trap cleanup EXIT
+
+      sudo rkt run \
+        --net=host \
+        --uuid-file-save="$uuid_file_save" \
+        --trust-keys-from-https \
+        --volume manifests,kind=host,source=/srv/kubernetes \
+        --mount volume=manifests,target=/srv/kubernetes \
+        {{.HyperkubeImageRepo}}:{{.K8sVer}} \
+        --exec /hyperkube -- kubectl create -f /srv/kubernetes/addons
```

Comment:
I'm wondering how exit codes will be handled here. For example, does `kubectl create` exit non-zero if it's trying to create assets that already exists (e.g. pretty much every reboot). And if it does exit non-zero, will `rkt run` exit non-zero causing this bash script to fail?